src/slg/scene/parsematerials.cpp

```diff
@@ -212,9 +212,13 @@ Material *Scene::CreateMaterial(const u_int defaultMatID, const string &matName,
 		} else if (props.IsDefined(propName + ".interiorior"))
 			interiorIor = GetTexture(props.Get(Property(propName + ".interiorior")(1.5f)));
 
-		const Texture *cauchyC = NULL;
-		if (props.IsDefined(propName + ".cauchyc"))
-			cauchyC = GetTexture(props.Get(Property(propName + ".cauchyc")(0.f, 0.f, 0.f)));
+		const Texture *cauchyB = NULL;
+		if (props.IsDefined(propName + ".cauchyb"))
+			cauchyB = GetTexture(props.Get(Property(propName + ".cauchyb")(0.f, 0.f, 0.f)));
+		// For compatibility with the past
+		else if (props.IsDefined(propName + ".cauchyc"))
+			SLG_LOG("WARNING: deprecated property " + propName + ".cauchyc");
+			cauchyB = GetTexture(props.Get(Property(propName + ".cauchyc")(0.f, 0.f, 0.f)));
```

Comment:
You need to put `{}` around this `else if` scope since it's not a single statement anymore.

Reply:
Good catch. Made a mistake copying the compiled binaries inot BlendLux and didn't notice...